Update monitor-tidb-cluster.md (#11654)
pingcap/docs: tidb-cloud/monitor-tidb-cluster.md

### Metrics on the cluster overview page

The cluster overview page provides general metrics of a cluster.

> For dedicated Tier:

Reviewer: ```suggestion
For Dedicated Tier clusters:

```

| Metric name  | Labels | Description                                   |
| :------------| :------| :-------------------------------------------- |
| QPS | total | The number of SQL statements executed per second in all TiDB instances. |

> | Query Duration | avg | The duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. |

Reviewer: ```suggestion
| Query Duration | avg | The duration from the time TiDB receives a client request to the time TiDB executes the request and returns the result to the client. |
```

| Active Connection | active connections | The number of active connections to all TiDB instances. |
| CPU | avg-tidb, avg-tikv, avg-tiflash | The average statistics of CPU usage for all TiDB instances. |
| Memory | avg-tidb, avg-tikv, avg-tiflash | TThe average statistics of Memory usage for all TiDB instances. |
| Used Storage Size | total-tikv, total-tiflash | The total storage size on TiKV or TiFlash instances. |

> 
> For Serverless Tier:

Reviewer: ```suggestion
For Serverless Tier:

```

| Metric name  | Labels | Description                                   |
| :------------| :------| :-------------------------------------------- |
| QPS | total | The number of SQL statements executed per second in Serverless Tier. |

> | Query Duration | avg | The duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. |

Reviewer: ```suggestion
| Query Duration | avg | The duration from the time TiDB receives a client request to the time TiDB executes the request and returns the result to the client. |
```

> | Active Connection | active connections | The number of active connections to Serverless Tier. |

Reviewer 2: ```suggestion
| Active Connection | active connections | The number of connections that are actively executing SQL statements to Serverless Tier. |
```

Reviewer 2: We realized that many people are confused with active connection and concurrent connection. So I suggest adding more information to let people understand the meaning of active connection.

> | Used Storage Size | total-tikv, total-tiflash | The total storage size on TiKV or TiFlash instances. |

Reviewer 2: We don't have total-tiflash at this time, but it will be available soon. Are we going to put it here right now, or create another PR when it's ready?

Another thing we need to be careful, there is another storage size gauge in clusters list page. We probably will have to figure out a way to add TiFlash there.

Author: OK. we can add it after released.

Author: I delete the description about tiflash

### Metrics on the cluster overview page

The cluster overview page provides general metrics of a cluster.

> For Dedicated Tier clusters:

Reviewer 3: ```suggestion
For TiDB Cloud Dedicated clusters:
```

| Metric name  | Labels | Description                                   |
| :------------| :------| :-------------------------------------------- |
| QPS | total | The number of SQL statements executed per second in all TiDB instances. |
| Query Duration | avg | The duration from the time TiDB receives a client request to the time TiDB executes the request and returns the result to the client. |
| Active Connection | active connections | The number of active connections to all TiDB instances. |
| CPU | avg-tidb, avg-tikv, avg-tiflash | The average statistics of CPU usage for all TiDB instances. |
| Memory | avg-tidb, avg-tikv, avg-tiflash | TThe average statistics of Memory usage for all TiDB instances. |

> | Used Storage Size | total-tikv, total-tiflash | The total storage size on TiKV or TiFlash instances. |

Reviewer: @lilyjazz Do you want to delete "tiflash" from here as well?

Author: no, dedicated tier has this label value "total-tiflash"

> For Serverless Tier:

Reviewer 3: ```suggestion
For TiDB Cloud Serverless clusters:
```

| Metric name  | Labels | Description                                   |
| :------------| :------| :-------------------------------------------- |

> | QPS | total | The number of SQL statements executed per second in Serverless Tier. |

Reviewer 3: ```suggestion
| QPS | total | The number of SQL statements executed per second in TiDB Cloud Serverless clusters. |
```
